packages/integration-sdk-cli/src/generator/template/src/config.ts.hbs

```diff
@@ -0,0 +1,57 @@
+import {
+  IntegrationExecutionContext,
+  {{#if configFields}}IntegrationValidationError,{{/if}}
+  IntegrationInstanceConfigFieldMap,
+  IntegrationInstanceConfig,
+} from '@jupiterone/integration-sdk-core';
+import { createAPIClient } from './client';
+
+/**
```

Comment:
Let's delete this and put this in `IntegrationInstanceConfigFieldMap` instead. Much better than copy/paste across so many repos.

Reply:
Same with below vars